x/subspaces/legacy/v2/keys.go

```diff
@@ -92,3 +92,11 @@ func GroupMemberStoreKey(subspaceID uint64, groupID uint32, user sdk.AccAddress)
 func UserPermissionStoreKey(subspaceID uint64, user sdk.AccAddress) []byte {
 	return append(PermissionsStoreKey(subspaceID), GetAddressBytes(user)...)
 }
+
+// SplitUserPermissionKey splits the provided UserPermissionStoreKey into the related subspace id and user address
+func SplitUserPermissionKey(key []byte) (subspaceID uint64, user sdk.AccAddress) {
+	key = key[1:]
+	subspaceID = GetSubspaceIDFromBytes(key[:9])
```

Comment:
```suggestion
	subspaceID = GetSubspaceIDFromBytes(key[:8])
```

Reply:
I realized it shouldn't matter, since the `GetSubspaceIDFromBytes` method uses the first 8 bytes (the amount required by `uint64` anyway). But I changed it anyway to make it look more correct